verification/negativeCases.go

```diff
@@ -0,0 +1,275 @@
+// Licensed under the Apache-2.0 license
```

Comment:
File that tests WrongLoclaity, invalid handle, Unsupported command and UnsupportedCommand flags